ci/bazelisk.sh

```diff
@@ -8,15 +8,15 @@
 # command-line specified .bazelrc (that is applied alongside //.bazelrc).
 
 if [[ -n "$PWD_OVERRIDE" ]]; then
-    cd "$PWD_OVERRIDE"
+    cd "$PWD_OVERRIDE" || exit
 fi
 
 echo "Running bazelisk in $(pwd)."
 
 # An additional bazelrc must be synthesized to specify precisely how to use the
 # GCP bazel cache.
 GCP_BAZELRC="$(mktemp /tmp/XXXXXX.bazelrc)"
-trap "rm $GCP_BAZELRC" EXIT
+trap 'rm $GCP_BAZELRC' EXIT
```

Comment:
Yes -- I thought about if either case would cause more surprising behavior if new code was added but ended up deciding that they're about the same. This is just to appease shellcheck.